src/object-cache.cls.php

```diff
@@ -154,7 +152,7 @@ private function debug_oc($text, $show_error = false)
 			return;
 		}
 
-		if (!$show_error && !$this->_cfg_debug) {
+		if (!$show_error && $this->_cfg_debug!=2) {
```

Comment:
We don't use 2
<img width="836" alt="image" src="https://github.com/user-attachments/assets/3b3ed753-e61f-4c03-bb05-00cc8a00e161" />